src/sikap/lib/lib-url.php

```diff
@@ -1,30 +1,40 @@
 <?php 
-	if (isset($_GET['page'])) {
-		$page = $_GET['page'];
-		switch ($page) {
-			case 'data':
-				require_once "data.php";
-				break;
+
+	if(isset($_GET['page'])){
+		$page = $_GET['page']; 
+	}else{
+		$page = "Page tidak diset di Method GET";
+	}
+	
+	
+	switch ($page) {
+/*		case 'data':
+			require_once "data.php";
+			break;
 
 			case 'data-add':
 				require_once "data-add.php";
 				break;
 
-			case 'data-edit':
-				require_once "data-edit.php";
-				break;
-				
-			case 'forgot':
-				require_once "forgot.php";
-				break;
-				
-			case 'reset':
-				require_once "reset.php";
-				break;
-				
-			default:
-				require_once "index-content.php";
-				break;
+		case 'data-edit':
+			require_once "data-edit.php";
+			break;
+*/		
+		case 'kehadiran':
+			require_once "kehadiran.php";
+			break;
+			
+		case 'forgot':
+			require_once "forgot.php";
+			break;
+			
+		case 'reset':
+			require_once "reset.php";
+			break;
+			
+		default:
+			require_once "index-content.php";
+			break;
 		}		
 	}
```

Comment:
Parse error: syntax error, unexpected '}', expecting end of file

Reply:
udah commit ulang yg resolve error bracket y

Reply:
masih error mba, saya yg fix aja ya